src/plugin.js

```diff
@@ -351,6 +378,18 @@ const initPlugin = function(player, options) {
       return;
     }
 
+    player.removeChild('BigPlayButton');
```

Comment:
we should null (or set to the vr bpb) out `player.bigPlayButton` as well and restore it on `reset`